src/app/context/AuthContext.tsx

```diff
@@ -27,6 +27,24 @@ interface AuthContextType {
 const AuthContext = createContext({} as AuthContextType);
 
 export function AuthProvider({ children }: { children: React.ReactNode }) {
+  //theme
+  useEffect(() => {
+    api.getSettings().then((response) => {
+      const settings = response;
```

Comment:
Cab it be worth typing this?

Reply:
You mean typing the response of the `getSettings` call?

Reply:
Yes, would that work?

Reply:
I think it would work but the variables that get sent from the API call are already typed so it might be redundant?